pootle_pytest/plugin.py

```diff
@@ -46,7 +48,8 @@
 PARAMETERS = (
     ("update_store_test_names", UPDATE_STORE_TESTS),
     ("file_import_failure_names", FILE_IMPORT_FAIL_TESTS),
-    ("wordcount_names", WORDCOUNT_TESTS))
+    ("wordcount_names", WORDCOUNT_TESTS),
+    ("unit_path_resolver_names", UNIT_PATH_RESOLVER_TESTS))
```

Comment:
Nit: We should go back to having the opening and closing brackets on its own line to avoid the diff to include stupid changes like this.